skimage/feature/_basic_features.py

```diff
@@ -3,7 +3,23 @@
 import numpy as np
 from skimage import filters, feature
 from skimage.util.dtype import img_as_float32
-from concurrent.futures import ThreadPoolExecutor
+from .._shared._dependency_checks import is_wasm
+
+if not is_wasm:
+    from concurrent.futures import ThreadPoolExecutor as PoolExecutor
+else:
+    from contextlib import AbstractContextManager
+
+    # Threading isn't supported on WASM, mock ThreadPoolExecutor as a fallback
+    class PoolExecutor(AbstractContextManager):
+        def __init__(self, *_, **__):
+            pass
```

Comment:
Is it necessary to define `__init__`?

Reply:
I don't think it should be needed – let me remove this.

Reply:
Addressed in 0bcf3c091c22715becb429b2e23af27c51c5d66f.

Reply:
Never mind, it does pass the number of workers as an argument, so it's needed: https://github.com/agriyakhetarpal/scikit-image/actions/runs/9385733448/job/25844607528